automotive/src/main/java/com/example/android/uamp/automotive/QrCodeSignInFragment.kt

```diff
@@ -39,34 +34,43 @@ import kotlinx.android.synthetic.main.qr_sign_in.toolbar
  */
 class QrCodeSignInFragment : Fragment(R.layout.qr_sign_in) {
 
+    private var binding: QrSignInBinding? = null
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        binding = QrSignInBinding.bind(view)
```

Comment:
If you make `binding` a local variable then it wouldn't be nullable (so you could get rid of many of the `?`s) and you wouldn't need to clear it in `onDestroy()`